README.md

```diff
@@ -86,7 +86,7 @@ A hardware TPM should always be used when real secrets and trust is required.
 
 ### Installer script
 
-Keylimes installer requires Python 3.6 or greater.
+Keylime installer requires Python version(>=3.6).
```

Comment:
I prefer the original wording here. Your suggestion puts the version number inside parenthesis, but the word "version" outside of it.

Reply:
Ok, makes sense I'll revert this to the original wording.